modules/haberdasher-labs_agreement_v0.1.0.json

```diff
@@ -1,4 +1,6 @@
 {
+  "id": "haberdasher-labs_agreement",
```

Comment:
I think we should keep the "eligibility" part of the name, ie `haberdasher-labs_agreement-eligibility`